.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: java
+
+jdk:
+  - oraclejdk8
+
+sudo: required
+dist: trusty
+
+services:
+  - docker
+
+before_install:
+  # This section can be removed once travis get equipped with docker 1.10 and docker-compose 1.6
```

Comment:
Is there an older version of docker on travis?
Or are we installing our own.
If the latter why cannot we install newer version?